projects/driver_controls/test/test_gpio_expander.c

```diff
@@ -0,0 +1,56 @@
+#include "unity.h"
+#include "log.h"
+#include "test_helpers.h"
+#include "gpio_expander.h"
+
+#define I2C_PORT_1 0
```

Comment:
Sure, but now you have multiple definitions of `I2C_PORT_1`. We don't support I2C on x86 anyways.